Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -349,6 +349,52 @@ public extension UIImage {
         self.init(data: data, scale: scale)
     }
 
+    #if !os(watchOS)
+    /// SwifterSwift: Create a new QR Code image from a string
+    /// 
+    /// - Parameters:
+    ///     -   qrCode: a `String`, representing the QR code image.
+    ///     -   correctionLevel: Controls the amount of additional data encoded to allow larger areas of the code to be damaged.
+    ///  There are four possible correction modes `L` 7% , `M` 15%, `Q` 25%, `H` 30%
+    convenience init?(qrCode: String, correctionLevel: String = "M") throws {
+        if correctionLevel != "L"
+            && correctionLevel != "M"
+            && correctionLevel != "Q"
+            && correctionLevel != "H" {
+            throw NSError(domain: "Invalid correction level. Only 4 possible correction modes L, M, Q, H", code: 0, userInfo: nil)
+        }
```

Comment:
I don't think you need to do this here. At most you can add an `assert`, but don't throw this error (which is wrongly created anyway).